lms/templates/courseware/accordion.html

```diff
@@ -25,7 +27,7 @@
         % for section in chapter['sections']:
         <div class="menu-item ${'active' if 'active' in section and section['active'] else ''} ${'graded'  if 'graded' in section and section['graded'] else ''}">
             <a href="${reverse('courseware_section', args=[course_id, chapter['url_name'], section['url_name']])}">
-                <p>${section['display_name']} ${'<span class="sr"> current section</span>' if 'active' in section and section['active'] else ''}</p>
+                <p>${section['display_name']} ${'<span class="sr"> current section</span>' if 'active' in section and section['active'] else '' | n, unicode}</p>
```

Comment:
oh.  i see.  The HTML is embedded inside the python string.  In that case, this line should be updated so that `current section` is translated using '_'.